docs/chapters/01.txt

```diff
@@ -270,10 +266,10 @@ beyond Core SQL conformance.
 The SQL Object Identifier identifies the characteristics of an
 SQL DBMS to other entities in an open systems environment. (The
 same information is available to SQL-data users in the
-INFORMATION_SCHEMA.SQL_LANGUAGES View.) Listing 1.1 shows
+``INFORMATION_SCHEMA.SQL_LANGUAGES`` View.) Listing 1.1 shows
 the required syntax for the SQL Object Identifier.
 
-Listing 1.1 Required Syntax for the SQL Object Identifier
+**Listing 1.1 Required Syntax for the SQL Object Identifier**
 
 ::
```

Comment:
for the text in this literal section, can you remove the blank lines between each line of code, so that it matches the display in the book

Reply:
so for example, at the moment you have this:

```
    <SQL Object Identifier> ::= <SQL provenance> <SQL variant>

         <SQL provenance> ::= <arc1> <arc2> <arc3>

              <arc1> ::= iso | 1 | iso(1)

              <arc2> ::= standard | 0 | standard(0)

              <arc3> ::= 9075

         <SQL variant> ::= <SQL edition> <SQL conformance>
...
```

but it should be this


```
    <SQL Object Identifier> ::= <SQL provenance> <SQL variant>
         <SQL provenance> ::= <arc1> <arc2> <arc3>
              <arc1> ::= iso | 1 | iso(1)
              <arc2> ::= standard | 0 | standard(0)
              <arc3> ::= 9075
         <SQL variant> ::= <SQL edition> <SQL conformance>
...
```

Reply:
this still needs changing

Reply:
At my computer this looks like it should be - there are no blank lines between
https://www.dropbox.com/s/2zuwv5j2o5u68bj/Screenshot%202018-10-05%2012.18.32.png?dl=0